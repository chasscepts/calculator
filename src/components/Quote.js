import AppHeader from './AppHeader';

const quote = 'Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“ William Paul Thurston';

export default function Quote() {
  return (
    <>
      <AppHeader />
      <p className="content">{quote}</p>
    </>
  );
}
