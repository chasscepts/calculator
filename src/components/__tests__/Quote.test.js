import React from 'react';
import renderer from 'react-test-renderer';
import ComponentWrapper from '../test_helpers/ComponentWrapper';
import Quote from '../Quote';

it('renders correctly', () => {
  const tree = renderer
    .create(<ComponentWrapper component={Quote} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
