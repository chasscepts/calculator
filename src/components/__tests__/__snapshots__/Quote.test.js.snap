// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <div
    style={
      Object {
        "display": "flex",
        "justifyContent": "space-between",
        "padding": "15px 30px",
      }
    }
  >
    <h1>
      Math Magicians
    </h1>
    <nav>
      <ul
        style={
          Object {
            "display": "flex",
            "listStyle": "none",
          }
        }
      >
        <li
          style={
            Object {
              "padding": "0 10px",
            }
          }
        >
          <a
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li
          style={
            Object {
              "borderLeft": "1px solid #ddd",
              "borderRight": "1px solid #ddd",
              "padding": "0 10px",
            }
          }
        >
          <a
            href="/calculator"
            onClick={[Function]}
          >
            Calculator
          </a>
        </li>
        <li
          style={
            Object {
              "padding": "0 10px",
            }
          }
        >
          <a
            href="/quote-of-the-day"
            onClick={[Function]}
          >
            Quote
          </a>
        </li>
      </ul>
    </nav>
  </div>,
  <p
    className="content"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“ William Paul Thurston
  </p>,
]
`;
